opcua/adapter.go

```diff
@@ -132,7 +134,26 @@ func (as *adapterService) ConnectThing(ctx context.Context, chanID, thingID stri
 }
 
 func (as *adapterService) Browse(ctx context.Context, serverURI, namespace, identifier string) ([]BrowsedNode, error) {
-	nodeID := fmt.Sprintf("%s;%s", namespace, identifier)
+	var nodeID string
```

Comment:
@WashingtonKK 

At present , for browser request we have only Namespace and Identified ,
```go
type browseReq struct {
	ServerURI  string
	Namespace  string
	Identifier string
}
```
https://github.com/absmach/magistrala/blob/main/opcua/api/requests.go#L8-L12

Here Identifier is Node ID , In OPCUA Node ID can be integer, string , GUID or base64String of binary data   

In present version supports it supports only integer type

It is good to have support for different types , But  it can't be in config, 

Because server can have mixed type of Node ID

So better to have it in request  
```go
ype browseReq struct {
	ServerURI  string
	Namespace  string
	Identifier string
	IdentifierType string
}
```